Utilities/Mergers/src/IntegratingMerger.cxx

```diff
@@ -35,12 +32,12 @@ IntegratingMerger::IntegratingMerger(const MergerConfig& config, const header::D
   : mConfig(config),
     mSubSpec(subSpec)
 {
-  mCollector = monitoring::MonitoringFactory::Get("infologger:///debug?qc");
-  //  mCollector->enableProcessMonitoring();
 }
 
 void IntegratingMerger::init(framework::InitContext& ictx)
 {
+  mCollector = monitoring::MonitoringFactory::Get(mConfig.monitoringUrl);
+  // mCollector->addGlobalTag(monitoring::tags::Key::Subsystem, monitoring::tags::Value::Mergers); // new Monitoring tag needed
```

Comment:
I presume that you asked for this new tag ?

Reply:
I need a PR merged even before that https://github.com/AliceO2Group/Monitoring/pull/248